src/applications/personalization/dashboard/components/FormItem.jsx

```diff
@@ -184,7 +184,7 @@ class FormItem extends React.Component {
 
 FormItem.propTypes = {
   savedFormData: PropTypes.object.isRequired,
-  toggleModal: PropTypes.func.isRequired,
+  toggleModal: PropTypes.func,
```

Comment:
moving forward the `toggleModal` will not be required. Actually soon the `toggleModal` won't be used at all. But for now we just need it to be optional.